src/components/groups/CreateGroup.js

```diff
@@ -199,58 +160,37 @@ const CreateGroup = props => {
 						onChange={handleChange}
 						value={values.acronym || ""}
 						name="acronym"
-						type="text"
-					/>
+						type="text" />
 				</Form.Group>
 				<Form.Field
 					required
 					label="Privacy Setting"
 					onChange={handleChange}
 					name="privacy_setting"
 					control="select"
-					defaultValue={values.privacy_setting || ""}
-				>
-					{window.location.pathname === "/editgroup" ? (
-						<option value={values.privacy_setting}>{privacy} </option>
-					) : (
-							<option value="" disabled hidden>
-								Choose Privacy setting...
-						</option>
-						)}
-					{privacy !== "Public" && privacy !== undefined ? (
-						<option value="public">Public</option>
-					) : null}
-					{privacy !== "Private" && privacy !== undefined ? (
-						<option value="private">Private</option>
-					) : null}
-					{privacy !== "Hidden" && privacy !== undefined ? (
-						<option value="hidden">Hidden</option>
-					) : null}
+					defaultValue={values.privacy_setting || ""}>
+
+					<option value={values.privacy_setting}>{privacy || ""}</option>
```

Comment:
I feel like this whole setup could use some explanatory comments. The dropdown works well, but the options do jump around depending on what you have selected, which is a little jarring. Definitely a very minor issue, but explaining why it's set up like this would be nice.

Reply:
Good point. I will comment that. There are a few things I need to go back and explain.